packages/eas-cli/src/update/utils.ts

```diff
@@ -47,14 +47,14 @@ export function getPlatformsForGroup({
 }
 
 export function formatPlatformForUpdateGroup(
-  updateGroup: {
+  updateGroup?: {
```

Comment:
then, with the changes above, you should be able to revert the changes to this file